run-make-check.sh

```diff
@@ -37,10 +33,14 @@ function run() {
     # increase the aio-max-nr, which is by default 65536. we could reach this
     # limit while running seastar tests and bluestore tests.
     local m=16
-    if [ $(nproc) -gt $m ]; then
-        m=$(nproc)
+    local procs="$(($(get_processors) * 2))"
+    if [ "${procs}" -gt $m ]; then
+        m="${procs}"
+    fi
+    local aiomax="$((65536 * procs))"
+    if [ "$(/sbin/sysctl -n fs.aio-max-nr )" -lt "${aiomax}" ]; then
+        $DRY_RUN sudo /sbin/sysctl -q -w fs.aio-max-nr="${aiomax}"
```

Comment:
Once we move to containers, we should remove this... I'd be worried if some building script would mess with my sysctl settings (and that's only for the testing stage). In fact, we should be able to run these scripts without sudo.

Reply:
Yep. One thing I almost did for this series but skipped for now was to replace calls to `sudo` to a function like `try_sudo` which checks the EUID by default and can be forced on/off by a variable. Again, to keep the series smaller and easier to review I defered that until later. As for this particular line, I agree that it's not good to do inside the container... but I left it for now.